package.json

```diff
@@ -1,19 +1,22 @@
 {
   "name": "vultr-action",
-  "version": "1.15",
+  "version": "2.0",
   "scripts": {
     "build": "ncc build src/index.ts -o dist --license licenses.txt"
   },
+  "engines": {
+    "pnpm": ">=8.0.0 <9.0.0",
+    "node": ">=20.0.0 <21.0.0"
+  },
   "devDependencies": {
-    "@types/node": "^16.11.10",
-    "@vercel/ncc": "^0.29.2",
-    "esbuild": "^0.12.29",
-    "esbuild-runner": "^2.2.1",
-    "typescript": "^4.5.2"
+    "@tsconfig/node20": "^20.1.4",
+    "@types/node": "^20.12.10",
+    "@vercel/ncc": "^0.38.1",
+    "typescript": "^5.4.5"
   },
   "dependencies": {
-    "@actions/core": "^1.6.0",
-    "@actions/github": "^5.0.0",
-    "axios": "^0.21.4"
+    "@actions/core": "^1.10.1",
```

Comment:
does the separation of what's in `dependencies` vs. `devDependencies` make sense?